src/components/feed/Modal/FeedCreateModal.tsx

```diff
@@ -61,11 +61,14 @@ function FeedCreateModal({ isModalOpened, meetingId, handleModalClose }: CreateM
     onSuccess: res => {
       queryClient.invalidateQueries(['getPosts']);
       alert('피드를 작성했습니다.');
-      mutatePostPostWithMention({
-        postId: res.postId,
-        orgIds: parseMentionedUserIds(formMethods.getValues().contents),
-        content: formMethods.getValues().contents,
-      });
+      const mentionedOrgIds = parseMentionedUserIds(formMethods.getValues().contents);
+      if (mentionedOrgIds.length > 0) {
+        mutatePostPostWithMention({
+          postId: res.postId,
+          orgIds: parseMentionedUserIds(formMethods.getValues().contents),
```

Comment:
```suggestion
          orgIds: mentionedOrgIds,
```

위와 같이 불필요한 함수 호출을 줄일 수 있을 것 같아요!
처음이라 코드 파악에 시간이 오래 걸렸을 텐데 빠르게 잘 처리해주셔서 감사합니다 :)